src/components/Footer/index.tsx

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
+import { Link } from 'react-router-dom'
 import './styles.scss';
 
 export default function Footer() {
-  return <div className="footer-container">Footer</div>;
+  return (
+    <div className="footer-container">
+      Footer
+      <div className="footer-item">
```

Comment:
There are no such `footer-container` and `footer-item`. If you see `Footer/styles.scss`, there's only `.footer` and `footer__item`. 🧐